gui/static/js/components/controlpanel.js

```diff
@@ -58,68 +74,122 @@ export default class ControlPanel extends React.Component {
             this.startLogging();
         }
         else {
-            console.log("Clicked on a direction button but nothing has been executed.");
+            axios({
+                method:'POST',
+                url:'/sendKV',
+                data: JSON.stringify({
+                    key: document.getElementById('kv_key').value,
+                    value: document.getElementById('kv_value').value,
+                    name: this.getBotID() //TODO
```

Comment:
What are all of these `// TODO`s after `this.getBotID()`s?

Reply:
removed